python/mlp/viewer/display_tools.py

```diff
@@ -22,17 +22,18 @@ def displaySphere(viewer, pos, size=0.01, color=[0, 0, 0, 1]):
     rootName = "s"
     # add indices until the name is free
     list = viewer.client.gui.getNodeList()
-    i = 0
-    name = rootName
-    while list.count(name) > 0:
-        name = rootName + "_" + str(i)
-        i += 1
-    viewer.client.gui.addSphere(name, size, color)
-    viewer.client.gui.addToGroup(name, viewer.sceneName)
-    #viewer.client.gui.setVisibility(name,'ALWAYS_ON_TOP')
-    q = pos + [0, 0, 0, 1]
-    viewer.client.gui.applyConfiguration(name, q)
-    viewer.client.gui.refresh()
+    if list is not None:
```

Comment:
`if list:` is enough

Reply:
btw, `list` is a [build-in function](https://docs.python.org/3/library/functions.html) name, please use something else.